### PR TITLE
Add Dockerfile and Makefile to build docker images

### PR DESCRIPTION
**Pull request by ankushagarwal**
Fixes https://github.com/kubeflow/kubeflow/issues/1055

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1122)

### CODE REVIEW

File: components/gcp-click-to-deploy/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+IMG = gcr.io/kubeflow-images-public/gcp-click-to-deploy
+
+TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)-$(shell git diff | shasum -a256 | cut -c -6)
```
Comment (jlewi) [inline components/gcp-click-to-deploy/Makefile]: Remove the "--tags" flag.

Comment (ankushagarwal) [reply components/gcp-click-to-deploy/Makefile]: Done

**Pull request merged**